### ISSUE TITLE
Accessibility on popups

### ISSUE THREAD

**Issue by MarMarIV** [xamarin/Xamarin.Forms#3612]
### Description
Accessibility does not work correctly. When the popup appears, the focus remains on the page behind the popup. When you click on an element, it focuses properly, but as soon as the element passes to the end, the focus moves back to the page behind the popup

### Steps to Reproduce

1. Turn on VoiceOver / Talkback
2. Set AutomationProperties.IsInAccessibleTree="true" and AutomationProperties.Hint on element in popup page
3. Open popup window and try navigate across elements on popup page

### Expected Behavior
On page appear focus on first element
Stay focus on page when escaping last element

### Actual Behavior
No focus on page appear
Leaving focus from page after last element

### Basic Information

- Version with issue: 3.0.0.482510
- Last known good version: 
- IDE: VS 2017 (15.8)
- Platform Target Frameworks: 
  - iOS:  11.4
  - Android: 8.1 
  - UWP: 16299

**Comment by hartez** [comment]
I believe this is a duplicate of #3622.

**Comment by samhouts** [comment]
closing as duplicate of #3622

**Comment by ncunning** [comment]
Has this been resolved or just closed? We are having this same issue

**Comment by hartez** [comment]
@ncunning Resolved - see #3622.
